pkg/check/pushsync/check_chunks.go

```diff
@@ -53,22 +55,26 @@ func CheckChunks(c *bee.Cluster, o Options) (err error) {
 
 			fmt.Printf("Node %s. Chunk %d found in the closest node %s Chunk: %s\n", nodeName, j, closestAddress.String(), ref.String())
 
-			// chunk should be replicated at least once in the neighborhood
-			closestName, closestAddress, err = chunk.ClosestNodeFromMap(overlays, closestAddress)
-			if err != nil {
-				return fmt.Errorf("node %s: %w", nodeName, err)
-			}
-			synced, err = ng.NodeClient(closestName).HasChunk(ctx, ref)
-			if err != nil {
-				return fmt.Errorf("node %s: %w", nodeName, err)
+			skipPeers := []swarm.Address{closestAddress}
+			// chunk should be replicated at least once either during forwarding or after storing
+			for range overlays {
+				name, address, err := chunk.ClosestNodeFromMap(overlays, skipPeers...)
+				skipPeers = append(skipPeers, address)
+				if err != nil {
+					continue
+				}
+				synced, err = ng.NodeClient(name).HasChunk(ctx, ref)
+				if err != nil {
+					continue
+				}
+				if synced {
```

Comment:
in a small cluster this can easily end up as the actual uploader node. can we put the uploader overlay in the skip peers?

Reply:
ok

Reply:
then, as an observation, with this requirements, the cluster size cannot ever been less than three